BepressImportDom.inc.php

```diff
@@ -755,7 +797,7 @@ function _cleanupFailure() {
 		}
 
 		foreach ($this->_errors as $error) {
-			if (size($error) > 1) {
+			if (sizeof($error) > 1) {
```

Comment:
If you'd like to remove the if altogether, this should be ok: `__($error[0], $error[1] ?? []);`